pkgs/development/python-modules/pytest-rerunfailures/default.nix

```
@@ -1,23 +1,23 @@
-{ lib, buildPythonPackage, pythonOlder, fetchPypi, pytest, mock }:
+{ lib, pythonPackages, pythonOlder, fetchPypi, setuptools, pytest, pytestCheckHook, mock }:
 
-buildPythonPackage rec {
+pythonPackages.buildPythonPackage rec {
```

```suggestion
buildPythonPackage rec {
```

We don't use `pythonPackages` for python libraries.